src/test/java/org/assertj/core/api/SoftAssertionsTest.java

```diff
@@ -158,6 +158,34 @@ public void all_assertions_should_pass() {
     softly.assertAll();
   }
 
+  @Test
+  public void all_composed_assertions_should_pass() {
+    SoftAssertions delegate = new SoftAssertions();
+    delegate.assertThat(1).isEqualTo(1);
+    delegate.assertAll();
+
+    softly.assertThat(1).isEqualTo(1);
+    softly.assertThat(Lists.newArrayList(1, 2)).containsOnly(1, 2);
```

Comment:
Minor: use AssertJ `org.assertj.core.util.List.list` and static import for readability.

Reply:
Done. 
In this case we also could have used Arrays.asList()